src/layouts/MainLayout.vue

```diff
@@ -241,6 +244,7 @@ export default defineComponent({
                     label: "Light / Dark",
                     action: () => {
                         this.$q.dark.toggle();
+                        console.info("Toggle Dark");
```

Comment:
We probably wanna use the Log library for this.

Reply:
Change to use the Log library. 
I also made sure the only reference to "console.log" is in the Log library.